flecsi-sp/burton/burton_corner.h

```diff
@@ -66,8 +67,153 @@ class burton_corner_t
   burton_corner_t( burton_corner_t && ) = delete;
   burton_corner_t & operator=( burton_corner_t && ) = delete;
 
+};
+
+////////////////////////////////////////////////////////////////////////////////
+//! \brief The burton_corner_t type provides an interface for managing and
```

Comment:
Extra "and"

Reply:
Good catch.  I'll be doing another PR shortly that refactors this file, and I'll fix this then.